eng/config/PublishData.json

```diff
@@ -44,7 +44,16 @@
       ],
       "vsBranch": "rel/d17.8",
       "vsMajorVersion": 17,
-      "insertionTitlePrefix": "[17.8P3]"
+      "insertionTitlePrefix": "[17.8]"
+    },
+    "release/dev17.9": {
+      "nugetKind": [
+        "Shipping",
+        "NonShipping"
+      ],
+      "vsBranch": "main",
```

Comment:
From my understanding, we want to still insert razor's `release/dev17.9` into VS's `main` until VS snaps. Is that correct?

Reply:
Correct.  Main, at this point, is for changes targeting the next preview of 17.9.